apps/loader/loader.js

```diff
@@ -114,15 +114,23 @@ function handleDownload(id) {
   var fileName='';
   store.getSlide(id)
       .then((response) => {
-        return response[0]['location'];
+        if (response[0]['location']) {
+          return response[0]['location'];
+        }
```

Comment:
@akhil-rana 
This is for the case when response[0] is undefined, accessing response[0]['location'] would give a reference error.
```suggestion
        if (response[0]) {
          return response[0]['location'];
        }
        else 
        {
         throw new Error('Slide not found');
        }
```

Reply:
I'm sorry but I don't understand that when will `response[0]` be undefined.
In case of `id` can't be found, error will be thrown in the fetch itself so.

Reply:
Okay, I got it. 
Thanks for the changes.

Reply:
Actually, the response is an empty array, when id of a non-existing slide is provided, in handleDownload()
*Screenshot*
![Screenshot from 2020-04-08 12-56-27](https://user-images.githubusercontent.com/45796806/78756576-a5179f00-7998-11ea-8463-c3f62c0d0877.png)

Reply:
Yes, I somehow missed that. Thanks.